titus-common/src/main/java/com/netflix/titus/common/util/grpc/reactor/GrpcToReactorServerFactory.java

```diff
@@ -22,4 +22,6 @@
 public interface GrpcToReactorServerFactory {
 
     <REACT_SERVICE> ServerServiceDefinition apply(ServiceDescriptor serviceDefinition, REACT_SERVICE reactService);
+
+    <REACT_SERVICE> ServerServiceDefinition apply(ServiceDescriptor serviceDefinition, REACT_SERVICE reactService, Class<REACT_SERVICE> reactorDetailedFallbackClass);
```

Comment:
Could you add a comment why we need `reactorDetailedFallbackClass`, and cannot just do `reactService.getClass()`?